doc/guides/west/build-flash-debug.rst

```diff
@@ -1,15 +1,18 @@
-.. _west-flash-debug:
+.. _west-build-flash-debug:
 
-Flashing and Debugging
-######################
+Building, Flashing and Debugging
+################################
 
-West provides three commands for running and interacting with Zephyr
-programs running on a board: ``flash``, ``debug``, and
+West provides four commands for building, flashing and interacting with Zephyr
```

Comment:
```suggestion
West provides four commands for building, flashing, and interacting with Zephyr
```

Reply:
It's actually five; attach was merged without documentation